Update docs for SEARCH operator (#1554)
neo4j/docs-cypher: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

label:updated[]
[source, cypher, role="noheader"]
----
MATCH (movie:Movie)
  SEARCH movie IN (
    VECTOR INDEX moviePlots
    FOR [1, 2, 3]
    WHERE movie.rating IN [7.6, 7.7, 8.0]
    LIMIT 10
  )
RETURN movie
----

> | The `WHERE` subclause used for xref:clauses/search.adoc#vector-search-filtering[vector search with filters] now supports the `IN` list operator.

Reviewer: ```suggestion
| `WHERE` predicates inside a xref:clauses/search.adoc#vector-search-filtering[vector `SEARCH`] now support the `IN` operator.
```
